app/controllers/games_controller.rb

```diff
@@ -0,0 +1,22 @@
+class GamesController < ApplicationController
+
+  rescue_from ActiveRecord::RecordNotFound, with: :game_not_found_error
```

Comment:
If it's not important to distinguish between different kinds of 404s, you could just have this rescue in ApplicationController and spare your individual controllers a lot of boilerplate.